extensions.json

```
@@ -93,6 +93,12 @@
       "repository": "https://github.com/appulate/vscode-file-watcher",
       "version": "1.0.3"
     },
+    {
+      "id": "arcanis.vscode-zipfs",
+      "repository": "https://github.com/yarnpkg/berry.git",
+      "version": "2.2.4",
```

When pinning a specific version, it's better to checkout the corresponding release tag or branch (as opposed to publishing straight from the default branch, which may contain unwanted/unfinished changes):

```suggestion
      "version": "2.2.4",
      "checkout": "vscode-zipfs/2.2.4",
```